packages/api/bin/serve.js

```diff
@@ -112,24 +95,6 @@ async function setTableEnvVariables(stackName) {
   };
 }
 
-// check if the tables and Elasticsearch indices exist
-// if not create them
-async function checkOrCreateTables(stackName) {
-  const tables = await setTableEnvVariables(stackName);
-  const limit = pLimit(1);
```

Comment:
I believe you've removed all references to pLimit/pRetry and those `require` statements should be removed

Reply:
Yup, good catch https://github.com/nasa/cumulus/pull/3354/commits/3acb8e0800a98d76983b3994c614bee95bb651c2